app/scripts/modules/core/src/securityGroup/AllSecurityGroupsCtrl.js

```diff
@@ -67,8 +67,13 @@ module.exports = angular
         updateSecurityGroups();
       };
 
+      function createSecurityGroupProviderFilterFn(application, account, provider) {
+        let sgConfig = provider.securityGroup;
```

Comment:
can `const` this since it won't be reassigned
```suggestion
        const sgConfig = provider.securityGroup;
```